seacrowd/sea_datasets/filipino_hatespeech_tiktok/filipino_hatespeech_tiktok.py

```diff
@@ -0,0 +1,131 @@
+_SEACROWD_VERSION = "1.0.0"
+
+_URLS = {
+    "raw": "https://raw.githubusercontent.com/imperialite/filipino-tiktok-hatespeech/main/data",
+}
+
+
+class FilipinoHateSpeechTikTok(datasets.GeneratorBasedBuilder):
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name="filipino_hatespeech_tiktok_source",
+            version=SOURCE_VERSION,
+            description="Filipino TikTok Hatespeech source schema",
+            schema="source",
+            subset_id="filipino_hatespeech_tiktok",
+        ),
+        SEACrowdConfig(
+            name="filipino_hatespeech_tiktok_seacrowd_text",
+            version=SEACROWD_VERSION,
+            description="Filipino TikTok Hatespeech Seacrowd schema",
+            schema="seacrowd_text",
+            subset_id="filipino_hatespeech_tiktok",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = "filipino_tiktok_hatespeech_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+        if self.config.schema == "source":
+            features = datasets.Features({"text": datasets.Value("string"), "label": datasets.Value("string")})
+        elif self.config.schema == "seacrowd_text":
+            features = schemas.text.features()
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        base_dir = _URLS["raw"]
+        data_files = {
+            "train": base_dir + "/train.csv",
+            "test": base_dir + "/test.csv",
+            "validation": base_dir + "/valid.csv",
+        }
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "filepath": data_files["train"],
+                    "split": "train",
+                },
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.TEST,
+                gen_kwargs={
+                    "filepath": data_files["test"],
+                    "split": "test",
+                },
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.VALIDATION,
+                gen_kwargs={
+                    "filepath": data_files["validation"],
+                    "split": "validation",
+                },
+            ),
+        ]
+
+    def _generate_examples(self, filepath: Path, split: str) -> Tuple[int, Dict]:
+        df = pd.read_csv(filepath).reset_index()
+        id = 0
+        if self.config.schema == "source":
+            for row in df.itertuples():
+                ex = {
+                    "text": row.text,
+                    "label": row.label if not pd.isna(row.label) else 0,
```

Comment:
This value becomes a float when I run the test script (`0.0` instead of `0`, `1.0` instead of `1`). I suggest turning this into an integer and then typecasting to string. 

```
Generating train split: 10038 examples [00:00, 38879.54 examples/s]
Generating test split: 4254 examples [00:00, 29701.82 examples/s]
Generating validation split: 4246 examples [00:00, 30438.29 examples/s]

INFO:__main__:Dataset sample [source]
{'text': 'Inaasahan na ni Vice President Jejomar Binay na may mga taong... https://t.co/SDytgbWiLh', 'label': '0.0'}
INFO:__main__:Dataset sample [seacrowd_text]
{'id': '0', 'text': 'Inaasahan na ni Vice President Jejomar Binay na may mga taong... https://t.co/SDytgbWiLh', 'label': 0}
INFO:__main__:Checking global ID uniqueness
```

Reply:
<img width="699" alt="image" src="https://github.com/SEACrowd/seacrowd-datahub/assets/75229742/631cdbf0-39e9-478f-9b4d-7d7d5cde5f82">
Thanks for the review! I typecasted the labels to string now